themes/scss/infowindow/cartodb-infowindow-default.scss

```diff
@@ -1,5 +1,5 @@
-@import '../variables/sizes';
-@import '../variables/mixins';
+@import '../../../node_modules/cartoassets/src/scss/variables/sizes';
```

Comment:
I wouldn't add non rendering scss on every file but just on the entry files, they are inherited